### PR TITLE
Bump year in license headers to 2021

### PR DESCRIPTION
**Pull request by vmax**
## What is the goal of this PR?

License headers are currently out of date (2020) for a while. This PR replaces 2020 with 2021 across all the files.

## What are the changes implemented in this PR?

* Replace `2020` with `2021` in source files

### CODE REVIEW

File: graql/reasoner/value-predicate.feature [hunk 1]
```diff
@@ -769,12 +769,12 @@ Feature: Value Predicate Resolution
       """
       insert
 
-      $x isa post, has creation-date 2020-07-01;
-      $x1 isa post, has creation-date 2020-07-02;
-      $x2 isa post, has creation-date 2020-07-03;
-      $x3 isa post, has creation-date 2020-07-04;
-      $x4 isa post, has creation-date 2020-07-05;
-      $x5 isa post, has creation-date 2020-07-06;
+      $x isa post, has creation-date 2021-07-01;
```
Comment (flyingsilverfin) [inline graql/reasoner/value-predicate.feature:772]: this intended @vmax ? probably is fnie haha

Comment (flyingsilverfin) [reply graql/reasoner/value-predicate.feature:772]: good to review your own PR before considering it done :)

File: graql/language/get.feature [hunk 2]
```diff
@@ -136,7 +136,7 @@ Feature: Graql Get Clause
       | colour        | string   | "blue"     | "green"          | "red"            | "yellow"   |
       | score         | long     | -38        | -4               | 18               | 152        |
       | correlation   | double   | -29.7      | -0.9             | 0.01             | 100.0      |
-      | date-of-birth | datetime | 1970-01-01 | 1999-12-31T23:00 | 1999-12-31T23:01 | 2020-02-29 |
+      | date-of-birth | datetime | 1970-01-01 | 1999-12-31T23:00 | 1999-12-31T23:01 | 2021-02-29 |
```
Comment (lolski) [inline graql/language/get.feature:139]: let's revert it

File: graql/language/match.feature [hunk 3]
```diff
@@ -1238,7 +1238,7 @@ Feature: Graql Match Query
       | calories    | long     | 1761       |
       | grams       | double   | 9.6        |
       | gluten-free | boolean  | false      |
-      | use-by-date | datetime | 2020-06-16 |
+      | use-by-date | datetime | 2021-06-16 |
```
Comment (lolski) [inline graql/language/match.feature:1241]: let's revert it

File: graql/language/match.feature [hunk 4]
```diff
@@ -1261,7 +1261,7 @@ Feature: Graql Match Query
...
```
Comment (lolski) [inline graql/language/match.feature:1264]: let's revert it

File: graql/language/match.feature [hunk 5]
```diff
@@ -1450,9 +1450,9 @@ Feature: Graql Match Query
     Given graql insert
       """
       insert
-      $x isa person, has name "Zoe", has age 21, has graduation-date 2020-06-01, has ref 0;
+      $x isa person, has name "Zoe", has age 21, has graduation-date 2021-06-01, has ref 0;
```
Comment (lolski) [inline graql/language/match.feature:1453]: let's revert it

File: graql/language/match.feature [hunk 6]
```diff
@@ -1450,9 +1450,9 @@ Feature: Graql Match Query
...
       $y (friend: $x) isa friendship, has age 21, has ref 1;
-      $z 2020-06-01 isa graduation-date, has age 21, has ref 2;
+      $z 2021-06-01 isa graduation-date, has age 21, has ref 2;
```
Comment (lolski) [inline graql/language/match.feature:1455]: let's revert it

File: graql/reasoner/value-predicate.feature (hunk 1 above)
Comment (lolski) [reply graql/reasoner/value-predicate.feature:772]: yes let's revert it

**Review by lolski — APPROVED** [review]

**Review by lolski — CHANGES_REQUESTED** [review]
See comments

File: graql/reasoner/value-predicate.feature (hunk 1 above)
Comment (vmax) [reply graql/reasoner/value-predicate.feature]: @flyingsilverfin @lolski addressed in https://github.com/graknlabs/behaviour/pull/136/commits/98f340f72238de28ce14dc2f629c82b1b1fc6242

File: graql/language/match.feature (hunk 3 above)
Comment (vmax) [reply graql/language/match.feature]: @flyingsilverfin @lolski addressed in https://github.com/graknlabs/behaviour/pull/136/commits/98f340f72238de28ce14dc2f629c82b1b1fc6242

File: graql/language/match.feature (hunk 5 above)
Comment (vmax) [reply graql/language/match.feature]: @flyingsilverfin @lolski addressed in https://github.com/graknlabs/behaviour/pull/136/commits/98f340f72238de28ce14dc2f629c82b1b1fc6242

File: graql/language/match.feature (hunk 6 above)
Comment (vmax) [reply graql/language/match.feature]: @flyingsilverfin @lolski addressed in https://github.com/graknlabs/behaviour/pull/136/commits/98f340f72238de28ce14dc2f629c82b1b1fc6242

**Review by lolski — APPROVED** [review]

**Review by flyingsilverfin — APPROVED** [review]

**Review by alexjpwalker — APPROVED** [review]

**Pull request merged**